src/main/java/com/cinema/Application.java

```diff
@@ -66,11 +68,11 @@ public static void main(String[] args) throws AuthenticationException {
 
         authenticationService.register(roma.getEmail(), roma.getPassword());
         User romaFromDB = authenticationService.login(roma.getEmail(), roma.getPassword());
-        shoppingCartService.addSession(session,romaFromDB);
-        shoppingCartService.addSession(secondSession,romaFromDB);
+        shoppingCartService.addSession(session, romaFromDB);
+        shoppingCartService.addSession(secondSession, romaFromDB);
         orderService.completeOrder(romaFromDB, shoppingCartService
                 .getByUser(romaFromDB).getTickets());
-        System.out.println("Empty cart : " + shoppingCartService.getByUser(romaFromDB));
-        System.out.println(orderService.getOrderHistory(romaFromDB));
+        logger.debug("Empty cart : " + shoppingCartService.getByUser(romaFromDB));
+        logger.debug(orderService.getOrderHistory(romaFromDB));
```

Comment:
I think Ksenia said in the Q&A that we can use logger.info in this project)

Reply:
Yep, I think it is better to use `info` level here
`debug` is for more fine-grained logging